mgmtsystem_action/views/mgmtsystem_action.xml

```diff
@@ -0,0 +1,194 @@
+                                    </div>
+                                </div>
+                            </div>
+                        </t>
+                    </templates>
+                </kanban>
+            </field>
+        </record>
+
+        <!-- Management System Action Report Pivot View -->
+
+                <record id="view_report_mgmtsystem_action_pivot" model="ir.ui.view">
+                    <field name="name">mgmtsystem.action.report.pivot</field>
+                    <field name="model">mgmtsystem.action.report</field>
+                    <field name="arch" type="xml">
+                        <pivot string="Actions" disable_linking="True">
+                            <field name="stage_id" type="row"/>
+                            <field name="user_id" type="col"/>
+                            <field name="number_of_actions" type="measure"/>
+                        </pivot>
+                    </field>
+                </record>
+
+        <!-- Management System Action Report Graph View -->
+
+                <record id="view_report_mgmtsystem_action_graph" model="ir.ui.view">
+                    <field name="name">mgmtsystem.action.report.graph</field>
+                    <field name="model">mgmtsystem.action.report</field>
+                    <field name="arch" type="xml">
+                        <graph string="Actions">
+                            <field name="stage_id"/>
+                            <field name="user_id"/>
+                            <field name="number_of_actions" type="measure"/>
+                        </graph>
+                    </field>
+                </record>
+
+        <!-- Management system Action Report Action -->
+
+                <record id="action_report_mgmtsystem_action" model="ir.actions.act_window">
+                    <field name="name">Management System Analysis</field>
+                    <field name="res_model">mgmtsystem.action.report</field>
+                    <field name="view_type">form</field>
+                    <field name="view_mode">pivot,graph</field>
+                    <field name="context">{"search_default_year":1,"search_default_User":1,"search_default_This Month":1,'group_by_no_leaf':1,'group_by':[]}</field>
+                    <field name="view_id"></field>  <!-- force empty -->
+                    <field name="help">Have a general overview of all actions processed in the system by sorting them with specific criteria.</field>
+                </record>
+
+
+
+
+        <record id="open_mgmtsystem_action_list" model="ir.actions.act_window">
+            <field name="name">Actions</field>
+            <field name="res_model">mgmtsystem.action</field>
+            <field name="view_type">form</field>
+            <field name="view_mode">tree,form,kanban</field>
+            <field name="view_id" ref="view_mgmtsystem_action_kanban"/>
+            <field name="search_view_id" ref="view_mgmtsystem_action_filter"/>
```

Comment:
It's more flexible if we don't provide specific views to use. You can remove the two lines above.